package.json

```diff
@@ -29,5 +29,11 @@
   },
   "browserslist": [
     "defaults"
-  ]
-}
+  ],
+  "science_history_institute_comment_about_engines" :
+    "We permit a range of node versions here:18.19 is GitHub CI. 20.9  is for everything else.",
+  "engines": {
+    "node": ">=18.19.0 <20.10.0",
```

Comment:
I wonder if it's safe and reliable to go all the way up to any `20.x`, ie less than 21?  To keep getting bugfixes and improvements without backwards breaking changes that will come in next major version? Assuming node is pretty safe on backwards breaking changes not happening unless major version?   It looks to me like the minor versions come pretty often, so I think maybe we do want this, not to require us to manually opt in to 20.11 20.12 etc, what do you think?

Reply:
Yeah, that's exactly my thinking. Based on your description of how we're using Node, I think we can rely on node working reliably for us, for all versions lower than `20.10`.

Reply:
Why not 20.10 and 20.11 (which are already out), and future versions less than 21?

Let's look at recent releaes history:

20.11.0: Jan 10, 2024
20.10.0:  Nov 22, 2023
20.9.0: Oct 24, 2023
20.8.0: 2023-09-28
20.7.0: 2023-09-18

It looks to me like the second digit (10.9, 10.10., 10.11) releases come out roughly monthly. I am suggesting maybe we we want to allow the app to use the new 20.x version (and have heroku install it) as available automatically, instead of having to change the package.json monthly to allow the new minor release?

So we can get these monthly minor releases with security fixes and improvemetns on heroku automatically as they come out? 

But require a manual change to package.json to upgrade to a `21` in the future, as it may have backwards-breaking changes. (Also 20.x is a "Long Term Support" release and 21 might not be, so we might not want it anyway). 

Right now, you are freezing it at 20.9, even though 20.10 and 20.11 are already out.

Reply:
Sorry, I misunderstood your comments. No problem at all - we now permit everything less than 21.0.0 .